src/CommonDBTM.php

```diff
@@ -2971,7 +2971,7 @@ public function isNewItem()
     /**
      * Check right on an item
      *
-     * @param integer $ID    ID of the item (-1 if new item)
+     * @param int|string $ID    ID of the item (-1 if new item)
```

Comment:
Are you sure of this ? It does not make sense.

```suggestion
     * @param int $ID    ID of the item (-1 if new item)
```

Reply:
Done in d83aea6135, but I had to remove the cleanID call then.